swf/src/write.rs

```diff
@@ -2148,11 +2164,8 @@ impl<W: Write> Writer<W> {
                 };
                 for glyph in &font.glyphs {
                     // Store offset for later.
-                    let offset = num_glyphs * 4 + shape_writer.output.len();
+                    let offset = (num_glyphs + 1) * 4 + shape_writer.output.len();
                     offsets.push(offset);
-                    if offset > 0xFFFF {
-                        has_wide_offsets = true;
-                    }
```

Comment:
Why remove this?

Reply:
I haven't implemented wide/standard offsets for font_version == 3